Documentation/cli-installation-scenarios.md

```diff
@@ -242,8 +262,16 @@ OSX/Linux:
 
 #### Windows obtain one-liner example 
 
+CMD:
+
+```
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex}"
+```
+
+PowerShell:
+
 ```
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1'))}"
+iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex
```

Comment:
@jasonwilliams200OK, Have you stepped through the script when it is failing for you?  I am curious if the file is actually getting downloaded.  In a working case it seems like `$Stream` would be set.  In a failing case, with `strict-mode` set, I agree that there is an issue with the script.  If `strict-mode` is not set, I wouldn't expect the error you mention.